packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -65,6 +99,28 @@ export const Council = () => {
             { label: 'Per Week', value: rewardPerDay.mul(asBN(7)) },
           ]}
         />
+
+        <StatisticItem>
+          <StatisticItemSpacedContent style={{ marginBottom: '8px' }}>
+            <StatisticLabel>Started At</StatisticLabel>
+
+            {council && (
+              <CustomBlockTimeWrapper>
+                <AboutText>{formatDateString(council.electedAt.timestamp)}</AboutText>
+                <BlockInfo block={council.electedAt} lessInfo={true} />
+              </CustomBlockTimeWrapper>
+            )}
+          </StatisticItemSpacedContent>
+          <StatisticItemSpacedContent>
+            <StatisticLabel>Ends At</StatisticLabel>
```

Comment:
```suggestion
            <StatisticLabel>Estimated end</StatisticLabel>
```
We cannot predict the exact end block/date due to multiple factors:
- block time is not always exactly 6s
- announcing period / elections can fail, in which case the council term extends until the next successful election